### PR TITLE
Update css_excludes.txt

### PR DESCRIPTION
**Pull request by svenms**
Updated Flatsome css ids that are created randomly

### CODE REVIEW

File: data/css_excludes.txt [hunk 1]
```diff
@@ -9,22 +9,34 @@
...
+cats-
+stack-
+timer-
+gap-
+gallery-
+portfolio-
+image_
+instagram-
+banner-
+banner-grid-
```
Comment (tynanbe) [inline data/css_excludes.txt:24]: This item is already matched by the previous line.

File: data/css_excludes.txt [hunk 2]
```diff
@@ -9,22 +9,34 @@
...
+map-
+text-
+text-box-
```
Comment (tynanbe) [inline data/css_excludes.txt:27]: This item is already matched by the previous line.

**Review by tynanbe — COMMENTED** [review]
Why remove all the id indicators (`#`)?

**Comment by hi-hai** [comment]
Closed due to missing response from OP.

**Pull request closed without merging**